crates/aptos/src/multisig/mod.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright © Aptos Foundation
+
+use crate::{
+    common::{
+        types::{TransactionOptions, TransactionSummary},
+        utils::profile_or_submit,
+    },
+    CliCommand, CliResult, CliTypedResult,
+};
+use aptos_types::account_address::AccountAddress;
+use async_trait::async_trait;
+use clap::{Parser, Subcommand};
+
+/// Tool for multisig account operations.
+#[derive(Debug, Subcommand)]
+pub enum MultisigTool {
+    Create(CreateMultisig),
+}
+
+impl MultisigTool {
+    pub async fn execute(self) -> CliResult {
+        match self {
+            MultisigTool::Create(tool) => tool.execute_serialized().await,
+        }
+    }
+}
+
+/// Create a multisig account.
+#[derive(Debug, Parser)]
+pub struct CreateMultisig {
+    /// Hex account address(es) to add as owners, each prefixed with "0x" and separated by spaces
+    #[clap(short, long, multiple(true), parse(try_from_str=crate::common::types::load_account_arg))]
+    pub additional_owners: Vec<AccountAddress>,
+    /// Number of signatures required to approve a transaction
+    #[clap(short, long)]
+    pub num_signatures_required: u64,
+    #[clap(flatten)]
+    pub txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for CreateMultisig {
+    fn command_name(&self) -> &'static str {
+        "CreateMultisig"
+    }
+
+    async fn execute(self) -> CliTypedResult<TransactionSummary> {
+        profile_or_submit(
```

Comment:
> I would check here, that your additional owners are not empty. Since the multisig needs more than one account.

I'm confused by this suggestion, as the Move logic has no requirements for multiple owners. For example see https://explorer.aptoslabs.com/txn/3202707?network=devnet , which successfully creates a single-owner multisig using tool in this PR.